scripts/fetch.py

```diff
@@ -26,7 +26,7 @@
 )
 parser.add_argument(
     "--xrd",
-    default=None,
+    default="root://cmsxrootd.fnal.gov//",
```

Comment:
suggest not to specify a global redirector, the default should always taken from specific site

Reply:
Hmm... Actually it did not work for me (brux) if I did not use a global redirector. I can try once more

Reply:
do you mean you don't have site_config in your `/cvmfs/cms.cern.ch/`?